platform-configs/configs-manager.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python 
```

Comment:
do we need this file?

Reply:
No, I forgot to delete it and must have did a `git add .`

Reply:
[9de6ce7](https://github.com/Dasharo/open-source-firmware-validation/pull/393/commits/9de6ce7ab3c284880e55685eed81f5946cc05f40)